src/llmcompressor/args/dataset_arguments.py

```diff
@@ -196,3 +197,10 @@ class DatasetArguments(CustomDatasetArguments):
             "definition"
         },
     )
+    model_input_device: Optional[str] = field(
```

Comment:
does inferring not work for deepseek? wondering if we can avoid another field that might confuse users

Reply:
cool, yeah if it's not needed for deepseek i think it's better to exclude for now, given the input arg bloat of `oneshot`. thanks!